### PR TITLE
Added the firstRunFlag feature

### LINKED ISSUE #301: Add a "first-run" query parameter optional feature
Add an option feature to Bubblewrap that allows it to append an extra parameter to `launchUrl` when the Trusted Web Activity is running for the first time. The value should be persisted and the parameter not attached / having a different value on the 2nd launch and onwards.

### PR DESCRIPTION
**Pull request by andreban**
-  When enabled, the `firstRunFlag` feature appends an extra
   query parameter to the the launch URL. The first time the app
   is launched, this parameter will contain the value `true`. On
   the following launches, the parameter will have the value
   `false`.
- This commit adds the feature, tests and modifies the template
  project
- Closes #301

### CODE REVIEW

File: packages/core/src/lib/features/FeatureManager.ts [hunk 1]
```diff
@@ -49,6 +52,10 @@ export class FeatureManager {
       this.addFeature(new AppsFlyerFeature(twaManifest.features.appsFlyer));
     }
 
+    if (twaManifest.features.firstRunFlag !== undefined) {
```
Comment (PEConn) [inline packages/core/src/lib/features/FeatureManager.ts:55]: I know this isn't the first PR with that pattern, but I feel a little weird that we're using an empty object to signify we should turn the feature on.

**Review by PEConn — APPROVED** [review]
Looks good, with one vague thought rather than a comment that needs change.

Also, I think it would be good if we had a test that took a config and compared the generated LauncherActivity.java against some golden.

File: packages/core/src/lib/features/FeatureManager.ts (hunk 1 above)
Comment (andreban) [reply packages/core/src/lib/features/FeatureManager.ts:55]: We could add a `enabled: boolean` to the config. The test would then become:

```ts
if (twaManifest.features.firstRunFlag && twaManifest.features.firstRunFlag.enabled)  {
```

Comment (PEConn) [reply packages/core/src/lib/features/FeatureManager.ts:55]: Yeah, that works well for me.

**Pull request merged**